### DISCUSSION TITLE
Does AutoGen Support MongoDB and SQL Agents Like LangChain?

### DISCUSSION THREAD

**Discussion by mohan-enmovil** [microsoft/autogen#4314] [Q&A]
Does AutoGen provide support for MongoDB and SQL agents, similar to LangChain, where I can interact with the database through chat?

**Comment by ekzhu** [comment]
Not yet. But you can use langchain agent inside autogen by wrapping it as a ChatAgent in 0.4

**Reply by NegiSushant** [reply]
could you share some stuff in autogen how can do this, so that we can pass NLP to the agent and agent get the data from the sql server based on the NLP user pass.
